js_modules/dagster-ui/packages/ui-core/src/runs/useRunsForTimeline.tsx

```diff
@@ -410,7 +410,8 @@ export const useRunsForTimeline = ({
     const earliest = jobs.reduce(
       (accum, job) => {
         const startTimes = job.runs.map((job) => job.startTime);
-        return {...accum, [job.key]: Math.min(...startTimes)};
+        accum[job.key] = Math.min(...startTimes);
```

Comment:
In retrospect this is pretty clown, its creating a new object on every iteration 🤯